test/functional/Java14andUp/playlist.xml

```diff
@@ -23,28 +23,51 @@
 -->
 
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TestConfig/playlist.xsd">
-      <test>
-          <testCaseName>Jep359Tests</testCaseName>
-          <variations>
-                  <variation>NoOptions</variation>
-          </variations>
-          <command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
-                  --enable-preview \
-                  -cp $(Q)$(LIB_DIR)$(D)asm-7.3.jar$(P)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(Q) \
-                  org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) -testnames Jep359Tests \
-                  -groups $(TEST_GROUP) \
-                  -excludegroups $(DEFAULT_EXCLUDE); \
-                  $(TEST_STATUS)
-          </command>
-          <levels>
-                  <level>sanity</level>
-          </levels>
-          <groups>
-                  <group>functional</group>
-          </groups>
-          <!-- Run for Java 14 only since this is a preview feature. -->
-          <subsets>
-                  <subset>14</subset>
-          </subsets>
-        </test>
+    <test>
```

Comment:
Is the whitespace change for Jep359Tests intentional?

Reply:
Yes. I found it was off when compared to other playlist files.